transports/websocket/src/framed.rs

```diff
@@ -120,22 +118,16 @@ where
         id: ListenerId,
         addr: Multiaddr,
     ) -> Result<(), TransportError<Self::Error>> {
-        let mut inner_addr = addr.clone();
-        let proto = match inner_addr.pop() {
-            Some(p @ Protocol::Wss(_)) => {
-                if self.tls_config.server.is_some() {
-                    p
-                } else {
-                    tracing::debug!("/wss address but TLS server support is not configured");
```

Comment:
Should we retain this log if we are listening on `/wss` to provide clarity?

Reply:
done